onecodex/analyses.py

```diff
@@ -367,7 +367,7 @@ def _to_classification_df(
 
             level = rank.level
             tax_ids_to_keep = []
-            unclassified_tax_ids = set()
```

Comment:
This needs to be a set because there could be duplicate tax IDs. To fix the pandas error, you can convert the set to a list on line 390 (the order of tax IDs doesn't matter because we're just summing their abundances)